src/Enzo/enzo_EnzoFieldAdaptor.hpp

```diff
@@ -0,0 +1,311 @@
+// See LICENSE_CELLO file for license and copyright information
+
+/// @file     enzo_EnzoFieldAdaptor.hpp
+/// @author   Matthew Abruzzo (matthewabruzzo@gmail.com)
+/// @date     2022-04-01
+/// @brief    [\ref Enzo] Declaration of the EnzoFieldAdaptor class
+///
+/// Adaptor used to provide a common interface for accessing field-like arrays
+/// from the Field class and EnzoEFltArrayMap.
```

Comment:
I think it would be useful to expand on this description a bit -- in particular to explain why this class is required and what parts of the code use it.